cap/weighting.py

```diff
@@ -352,3 +353,64 @@ def split_length(data: Caption,
         data[number-2].weight += 0.25 * factor
 
     return data
+
+
+def _abs_linspace(start: int, stop: int, step: int) -> List[float]:
```

Comment:
moet je hier niet ook nog Arg: en Returns: doen??

Reply:
Ik vond het niet echt nodig omdat het ten eerste geen publieke functie is (alleen wij hoeven 'm te gebruiken), en ten tweede is het duidelijk gebaseerd op numpy's linspace, die uitgebreide documentatie heeft